paymentsheet/src/test/java/com/stripe/android/link/LinkActivityContractTest.kt

```diff
@@ -114,9 +114,7 @@ class LinkActivityContractTest {
         return LinkActivityContract(
             nativeLinkActivityContract = nativeLinkActivityContract,
             webLinkActivityContract = webLinkActivityContract,
-            linkGateFactory = {
-                linkGate
-            }
+            linkGateFactory = FakeLinkGate.Factory(linkGate)
         )
```

Comment:
👀 Had to make this change because `LinkGate.Factory` is no longer a fun interface (pun absolutely intended).